X/XDiag/build_tarballs.jl

```diff
@@ -86,13 +86,13 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.7")),
+    BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.17")),
     Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7"); compat="0.13.2"),
```

Comment:
you may also wish to switch to a newer `libcxxwrap_julia_jll`, this may even be required for actual run-time Julia 1.12 compat (but then your Julia package also needs a newer CxxWrap.jl in its dependencies)

Reply:
Thanks. Which one should I choose then, I am not so familiar with all of this?

Reply:
@fingolfin I updated 'libcxxwrap_julia_jll' compat to 0.14.3 now. You say I then also need to update the CxxWrap dependency in the 'XDiag.jl' package. Should this then just be 0.17, do you know?